src/main/java/school/lemon/changerequest/java/introduction/hw1/Task3.java

```diff
@@ -2,6 +2,13 @@
 
 public class Task3 {
     public static boolean isBetween(int a, int b, int c) {
+        if (b >= a && b <= c || b <= a && b >= c) return true;
```

Comment:
It can be simpler, just: `return b >= a && b <= c || b <= a && b >= c;` because result of logical operations is boolean.